src/Microsoft.AspNet.Mvc.WebApiCompatShim/ApiController.cs

```diff
@@ -91,7 +91,7 @@ public HttpRequestMessage Request
         /// Gets a factory used to generate URLs to other APIs.
         /// </summary>
         /// <remarks>The setter is intended for unit testing purposes only.</remarks>
-        [Activate]
```

Comment:
Assuming you did a ctrl+shift+F for this, but if you haven't please do and review all the uses.